### DISCUSSION TITLE
GraphQL API for Cassandre made with DGS

### DISCUSSION THREAD

**Discussion by straumat** [Netflix/dgs-framework#653] [Show and tell]
[Casssandre](https://github.com/cassandre-tech/cassandre-trading-bot) is an open-source spring boot starter that allows you to quickly develop your crypto trading bot. We developed a graphql API thanks to dgs! Thx a lot !

**Comment by srinivasankavitha** [comment]
That's great to hear! Please let us know if there are any feature gaps/requests or issues that we can address to help make it better.
